config/config.play.js

```diff
@@ -8,16 +9,13 @@ module.exports = {
   },
   dist: 'dist-play',
   postcss: config.postcss,
-  templateCompiler: true,
-  webpack: {
-    resolve: {
-      modules: [
-        path.resolve('src')
-      ]
-    }
+  webpack (config) {
+    ['app', 'preview'].forEach(entry => {
```

Comment:
Here's a special keyword `[hot]` for this:

```js
{
  entry: ['[hot]', './play/app.js']
}
```

But I think I will rename it to `:hot:` which looks less confusing.